backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt

```diff
@@ -49,7 +53,11 @@ class DefaultDocumentComponent(
             .awaitSingle()
     }
 
-    override suspend fun getAll(types: List<DocumentType>, statuses: List<DocumentStatus>, pageable: Pageable): List<DocumentEntity> {
+    override suspend fun getAll(
+        types: List<DocumentType>,
+        statuses: List<DocumentStatus>,
+        pageable: Pageable
+    ): List<PersonDocumentEntity> {
```

Comment:
maybe use `PersonDocumentEntity` also to get a single object (the method above)?

and for `update` method?

Reply:
make in update